app/views/dashboard/_pending_nodes.html.slim

```diff
@@ -16,7 +16,7 @@
         p.empty-text
           | You currently have no nodes to be accepted for bootstrapping.
         .has-content.hidden
-          p Accepting nodes into the cluster might take a while.
+          p Accepting nodes into the cluster might take a while. Be aware that it's not possible to accept a new node while a node is being bootstrapped.
```

Comment:
I would make this a bit more clear from the wording, maybe `to accept a new node while another node is being bootstrapped`?